modules/platform/nuxeo-automation/nuxeo-automation-server/src/main/java/org/nuxeo/ecm/automation/server/jaxrs/batch/handler/impl/DefaultBatchHandler.java

```diff
@@ -55,4 +56,9 @@ public boolean completeUpload(String batchId, String fileIndex, BatchFileInfo fi
                 && Objects.equals(fileEntry.getBlob().getDigest(), fileInfo.getMd5());
     }
 
+    @Override
+    public Map<String, Object> refreshToken(String batchId) {
+        throw new UnsupportedOperationException("Refresh token is not supported for the default batch handler.");
```

Comment:
You should move this to a default method in the interface as others may have custom batch handlers. Adapt the message of course (the default -> this).

Reply:
[for the history of that specific ticket] It has been decided to use `default` only for backports.